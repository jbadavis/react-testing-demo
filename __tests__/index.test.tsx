import {render} from '@testing-library/react';
import App from '@/pages/index';

describe('App', () => {
  test('render correctly', () => {
    const {container} = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
