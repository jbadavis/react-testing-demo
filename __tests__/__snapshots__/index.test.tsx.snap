// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render 1`] = `
<div>
  <div
    class="chakra-stack css-1kkt86i"
  >
    <p
      class="chakra-text css-0"
    >
      Welcome to the Planes Internal Testing Demonstration.
    </p>
    <p
      class="chakra-text css-0"
    >
      This is a core module and must be passed before your yearly appraisal.
    </p>
  </div>
</div>
`;
